server/index.js

```diff
@@ -179,18 +179,6 @@ export default class Server {
         await serveStatic(req, res, path)
       },
 
-      '/_next/:buildId/page/_error*': async (req, res, params) => {
```

Comment:
We don't need this route.
This will be handled using the generic page resolving route.

Reply:
Exactly 👍